.pre-commit-config.yaml

```diff
@@ -11,11 +11,12 @@ repos:
     -   id: check-json
     -   id: check-merge-conflict
     -   id: requirements-txt-fixer
-  - repo: https://github.com/humitos/mirrors-autoflake
-    rev: v1.1
-    hooks:
-      - id: autoflake
-        args: ["-i", "--remove-all-unused-imports"]
+# FIXME fails for me with ModuleNotFoundError: No module named 'distutils'
```

Comment:
Hmm, works for me, so I guess I have it cached locally. I should replace this whole bit with ruff.